model/src/main/scala/cromwell/pipeline/model/validator/Wrapped.scala

```diff
@@ -16,34 +16,6 @@ trait Wrapped[T] extends Any {
   override def hashCode: Int = this.getClass.hashCode + unwrap.hashCode()
   override def toString: String = unwrap.toString
 }
-object Wrapped {
-  trait Companion {
-    type Type
-    type Error
-    type Wrapper <: Wrapped[Type]
-    type ValidationResult[A] = Validated[NonEmptyChain[Error], A]
-    implicit def wrappedOrdering(implicit ord: Ordering[Type]): Ordering[Wrapper] = Ordering.by(_.unwrap)
-    implicit def wrapperFormat(implicit show: Show[Error], format: Format[Type]): Format[Wrapper] =
-      new Format[Wrapper] {
-        override def reads(json: JsValue): JsResult[Wrapper] = from(
-          json.as[Type]
-        ) match {
-          case Valid(value)    => JsSuccess(value)
-          case Invalid(errors) => JsError(errors.toNonEmptyList.show)
-        }
-        override def writes(wrapped: Wrapper): JsValue = Json.toJson(wrapped.unwrap)
-      }
-    protected def create(value: Type): Wrapper
-    protected def validate(value: Type): ValidationResult[Type]
-    def from(value: Type): ValidationResult[Wrapper] =
-      validate(value).map(create)
-    final def apply(value: Type, evidence: Enable.Unsafe.type)(implicit show: Show[Error]): Wrapper =
-      validate(value) match {
-        case Valid(content)  => create(content)
-        case Invalid(errors) => throw new Enable.UnsafeException(errors.toNonEmptyList)
-      }
-  }
-}
 
 object Enable {
```

Comment:
I guess this object must be placed with `WrappedFactoryPattern` (`Wrapped.Companion`) trait.

Reply:
moved